ts/store/reducers/persistedPreferences.ts

```diff
@@ -46,7 +46,7 @@ export type PersistedPreferencesState = Readonly<{
   // changing the variable value later). Typescript cannot detect this so
   // be sure to handle such case when reading and using this value
   isDesignSystemEnabled: boolean;
-  isNewWalletSectionEnabled: boolean;
+  isNewScanSectionEnabled: boolean;
```

Comment:
```suggestion
  isNewScanSectionEnabled?: boolean;
```
I see we don't introduce this persisted state with a migration. What about marking it as nullable?